packages/core/src/feed.ts

```diff
@@ -0,0 +1,58 @@
+import { Subject, Observable, map, NextObserver } from 'rxjs'
+import { type StreamMetadata, type StreamState } from '@ceramicnetwork/common'
+import { CommitID } from '@ceramicnetwork/streamid'
+import { StreamUtils } from '../../common/lib/utils/stream-utils.js' //TODO: once commitIdFromStreamState is part of the package remove this line, and update the @ceramicnetwork/common import
+import type { AnchorStatus } from '@ceramicnetwork/common'
+
+export type Document = {
```

Comment:
to avoid confusion with the existing `Document` class in the http-client (which we should also probably rename), can we call this `FeedDocument` or `FeedEvent` or something like that?

Reply:
I agree it can be confusing, I'll go for `FeedDocument`.